cds/modules/theme/static/js/cds/module.js

```diff
@@ -576,3 +576,24 @@ app.directive('popover', function() {
       element.find('a[rel=popover]').popover();
   };
 });
+
+/**
+ * filter for shating on different social platforms
+ * more details about how we build the URLs you can find on: https://github.com/bradvin/social-share-urls
+ */
+app.filter('assembleShareURL', ['$window', function($window) {
+  return function(record, platform) {
+    if (!record) { return; }
+
+    var currentPageAddress = $window.location.href;
+    var platformMapping = {
+      'facebook': 'https://www.facebook.com/sharer.php?u=' + currentPageAddress,
```

Comment:
It looks like that for Facebook you can also add the param `t` for the title... even if not listed in the doc repo... to be checked

Reply:
I checked and it's not working :(. You can only add the url and that's all.